src/generate.tsx

```diff
@@ -187,7 +188,7 @@ export interface GenerateConfig<OptionsType extends object[]> {
   /** Convert single raw value into { label, value } format. Will be called by each value */
   getLabeledValue: GetLabeledValue<FlattenOptionsType<OptionsType>>;
   filterOptions: FilterOptions<OptionsType>;
-  findValueOption: // Need still support legacy ts api
+  findValueOption:  // Need still support legacy ts api
```

Comment:
No need 2 space comment

Reply:
I have removed the spaces but it seems that they are added by `prettier` in `preCommit` hook.